checklists/lz_checklist.en.json

```diff
@@ -1021,7 +1027,7 @@
     {
       "category": "Management",
       "subcategory": "Monitoring",
-      "text": "Use Update Management in Azure Automation as a long-term patching mechanism for both Windows and Linux VMs. ",
+      "text": "Use Update Management in Azure Automation?as a long-term patching mechanism for both Windows and Linux VMs. ",

```

Comment:
Same thing, could you please remove the `?` here?

Reply:
fixed